layout-server/src/main/java/org/finos/vuu/layoutserver/dto/response/ApplicationLayoutDto.java

```diff
@@ -0,0 +1,16 @@
+package org.finos.vuu.layoutserver.dto.response;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+
+@Data
+@RequiredArgsConstructor
+@AllArgsConstructor
+@Builder
```

Comment:
Are all of these required?

Reply:
I'm sure builder was at one point. The other two were required for some tests, but a quick refactor and I can get rid of them too.